mrc-ide/hint
# mrc-3118 Upload output zip action which calls hintr endpoint via backend

## Description

This PR loads the whole outputZip and prints response in console. I have added endpoints to the backend and call these endpoints in front end. 

TO TEST
* Use the old load menu, it does accept json and zip file.
* Once loaded see console for response from rehydrate.

## Type of version change
_Delete as appropriate_

Minor

## Checklist
